f8a_worker/storages/s3_userprofilestore.py

```diff
@@ -0,0 +1,11 @@
+"""Import AmazonS3"""
+from . import AmazonS3
+
+class S3UserProfileStore(AmazonS3):
+    """An Adapter Class to handle storage of User Profile information to S3"""
+    def store_in_bucket(self, content):
+        """Stores the User Profile information into S3 Buckets"""
+        email = content.get('email')
+        folder = 'new_user_profile'
+        key = folder + '/' + email + '.json'
```

Comment:
Yup, that should do the trick, otherwise there will be an error:

```python
>>> 'foo' + None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: cannot concatenate 'str' and 'NoneType' objects
```